ministryofjustice/correspondence_tool_staff
# Remove mimetype-fu gem and use Rack as alternative

## Description
Removes dependance on mimetype-fu and uses Rack alternative instead.

## Self-review checklist


### Screenshots


### Related JIRA tickets
[CT-3641](https://dsdmoj.atlassian.net/browse/CT-3641)

### Manual testing instructions
- There was only one place where this gem was used in the `show` action in the `AttachmentsController`.
- So try to view a preview of an upload as this should test this functionality.
